pixorama_flutter/.gitignore

```diff
@@ -31,12 +31,8 @@
 .pub/
 /build/
 
-android/
```

Comment:
Let's keep these in the .gitignore file. The platform specific files are hard to keep up-to-date and easy to recreate (just run `flutter create .`).

So, let's add these lines back, then remove the platform specific directories from the PR.

Reply:
Resolved